package.json

```diff
@@ -21,7 +21,7 @@
 		"test": "exit 0"
 	},
 	"dependencies": {
-		"@chasemoskal/magical": "^0.0.4",
+		"@chasemoskal/magical": "^0.0.5",
```

Comment:
be warned, you'll have to `npm install` beyond this point -- here's where we picked up that magical hotfix